src/nm-core-utils.c

```diff
@@ -2797,57 +2797,97 @@ nm_utils_file_get_contents (int dirfd,
 
```

Comment:
[...] "In case of error, *the we* won't be persisted" [...]
The commit msg probably has a typo.